contracts/AnchoredView/AnchoredView.sol

```diff
@@ -27,17 +27,17 @@ interface AnchorOracle {
  * @author Compound Labs, Inc.
  */
 contract AnchoredView is SymbolConfiguration {
-    /// @notice The mapping of posted by source prices per symbol
+    /// @notice The mapping of posted by reporter prices per symbol
```

Comment:
Isn't this just the mapping of (anchored, medianized, etc.) prices by symbol?

Reply:
tru